components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt.ui/src/main/resources/web/identity-claim-mgt/add-external-claim-finish-ajaxprocessor.jsp

```diff
@@ -45,7 +45,7 @@
     String cookie = (String) session.getAttribute(ServerConstants.ADMIN_SERVICE_COOKIE);
 
     String externalClaimDialectURI = request.getParameter("externalClaimDialectURI");
-    String externalClaimURI = request.getParameter("externalClaimURI");
+    String externalClaimURI = request.getParameter("externalClaimURI").trim();
```

Comment:
Shall we add a null check?

Reply:
Used `StringUtils` to trim the parameters.